ehitchhiking-ui/src/app/shared/components/rate-passengers-modal/rate-passengers-modal.component.ts

```diff
@@ -41,7 +41,7 @@ export class RatePassengersModalComponent implements OnInit {
 	loadPassengersList(): void {
 		this.loading = true;
 		this.apiRatePassengersService
-			.getTripPassengers(this.idTripPassenger)
+			.getTripPassengers(this.data.tripId)
```

Comment:
You don't commit file where tripId passed to the modal's data

Reply:
done